<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saf_maint_dialog</class>
 <widget class="QDialog" name="saf_maint_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SAF Maintenance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="base_path_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Base Path:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="browse_pushButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>SPRAT Analysis Folders (SAFs):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>Folder</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Rec Files
(MB)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Output Files
(MB)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>SPRAT Files
(MB)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Total
(MB)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Last Modified</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Clean</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Mothballed</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="sel_all_pushButton">
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sel_none_pushButton">
       <property name="text">
        <string>Select None</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clean_pushButton">
       <property name="text">
        <string>Clean</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mothball_pushButton">
       <property name="text">
        <string>Mothball</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_pushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
